barretenberg/cpp/src/barretenberg/relations/delta_range_constraint_relation.hpp

```diff
@@ -55,37 +55,29 @@ template <typename FF_> class DeltaRangeConstraintRelationImpl {
         auto delta_4 = w_1_shift - w_4;
 
         // Contribution (1)
-        auto tmp_1 = delta_1;
-        tmp_1 *= (delta_1 + minus_one);
-        tmp_1 *= (delta_1 + minus_two);
-        tmp_1 *= (delta_1 + minus_three);
+        auto tmp_1 = (delta_1 + minus_one).sqr() - one;
```

Comment:
why not `+minus_one` rather than `-one`

Reply:
maybe i'm brainded but i computed by hand the diff and i don't get the same result, can you please assure me I'm brainded?

Reply:
Replaced. wdym the result is not the same? (a+1)*(a-1)= a^2 - 1. We have a(a-1)(a-2)(a-3)= ((a-1)^2-1)((a-2)^2-1)

Reply:
I can't maths :)